packages/views/screens/managers/register/register.js

```diff
@@ -10,70 +8,44 @@ import BackupPassphrase from './backupPassphrase';
 import ConfirmPassphrase from './confirmPassphrase';
 import AccountCreated from './accountCreated';
 import styles from './register.css';
+import useCreateAccounts from '../../../../wallet/manager/useCreateAccounts';
```

Comment:
Very long relative imports like this need to be put above imports from the same folder because it leads to some lint errors